integrations/docker/images/chip-build-vscode/Dockerfile

```diff
@@ -4,6 +4,7 @@ FROM connectedhomeip/chip-build-efr32:${VERSION} AS efr32
 FROM connectedhomeip/chip-build-android:${VERSION} AS android
 FROM connectedhomeip/chip-build-esp32-qemu:${VERSION} as esp32
 FROM connectedhomeip/chip-build-mbed-os:${VERSION} AS mbedos
+FROM connectedhomeip/chip-build-tizen:${VERSION} AS tizen
```

Comment:
what about the "Get Tizen packagese"  step in the tizen images? Are those required for build?

if yes, we probably need them in the vscode image as well (unsure how to handle duplicates...should we put them in chip-build in general?) and if not, we should maybe remove them from the tizen image.

Reply:
Those are required for build. so the vscode image was added in 27 line as follows.
```
 COPY --from=tizen /opt/tizen_sdk /opt/tizen_sdk
```

Reply:
I am trying to validate that all steps are kind of here. Could you confirm each? I see:

- tizen user/group add (is that needed? Why not in vscode?)
- apt install - guessing this is just for download & decompress? Could you confirm that we do not need it here?

For the rest I saw rpm and somehow assumed they did not go to /opt/tizen (missed also that the usr paths did not start with /) so they are likely fine.

Reply:
> I am trying to validate that all steps are here or not actually needed. Could you confirm each? I see:
> 
>     * tizen user/group add (is that needed? Why not in vscode?)

It's a good practice to run containers with a non-root user as one way to mitigate the [CVE-2019-5736](https://blog.dragonsector.pl/2019/02/cve-2019-5736-escape-from-docker-and.html) so maybe this practice needs to be propagated to other Dockerfiles in a separated PR.

> 
>     * apt install - guessing this is just for download & decompress? Could you confirm that we do not need it here?
> 

I think that you're referring to the [obs-build](https://packages.ubuntu.com/focal/obs-build) and [cpio](https://packages.ubuntu.com/focal/cpio) packages which are required to run the `unrpm` command used in the line 69.


> 
> For the rest I saw rpm and somehow assumed they did not go to /opt/tizen (missed also that the usr paths did not start with /) so they are likely fine.

Reply:
> I think that you're referring to the obs-build and cpio packages which are required to run the unrpm command used in the line 69.

Thanks for your comment. actually, the packages installed as apt-get is for `unrpm` command execution. and tizen packages are copied from tizen docker to vscode docker `/opt/tizen_sdk` path.